src/main/java/uk/gov/hmcts/dts/fact/services/admin/list/AdminLocalAuthorityService.java

```diff
@@ -43,9 +48,16 @@ public LocalAuthority updateLocalAuthority(final Integer localAuthorityId, final
         checkIfLocalAuthorityAlreadyExists(localAuthorityId, name);
 
         // Change local authority entity name
+        final List<LocalAuthority> originalList = getAllLocalAuthorities();
         final uk.gov.hmcts.dts.fact.entity.LocalAuthority existingEntity = localAuthorityEntity.get();
         existingEntity.setName(name);
-        return new LocalAuthority(localAuthorityRepository.save(existingEntity));
+        LocalAuthority newLocalAuthority = new LocalAuthority(localAuthorityRepository.save(existingEntity));
+        adminAuditService.saveAudit(
+            AuditType.findByName("Update local authority"),
```

Comment:
This is probably due to my lack of Java knowledge but why can't we call AuditType.UPDATE_COURT_LOCAL_AUTHORITIES rather than passing a string to findByName?

Reply:
It's based on the other examples on the service for how we search for enums. I was tempted to do it in the way you mentioned, but I think I leaned on Kian and the original dev's approach. May be best to keep it the same for consistency